src/Management/src/Endpoint/CloudFoundry/SecurityUtils.cs

```diff
@@ -16,15 +16,14 @@ internal sealed class SecurityUtils
 {
     public const int DefaultGetPermissionsTimeout = 5000; // Milliseconds
     public const string ApplicationIdMissingMessage = "Application id is not available";
-    public const string EndpointNotConfiguredMessage = "Endpoint is not available";
+    public const string EndpointNotConfiguredMessage = "EndpointHandler is not available";
```

Comment:
In this case, I feel the class name helps someone trouble shoot faster than mentioning the now abstract "Endpoint"

Reply:
I'm not deep in the context on this one.. Is it likely to be seen/used by a Steeltoe developer or consumer? Are we going to expose Steeltoe users to the handlers, or are they really only an internal detail? It might be simpler if our users aren't aware there are things called handlers at all